src/vs/vscode.proposed.d.ts

```diff
@@ -1048,16 +1048,35 @@ declare module 'vscode' {
 
 	//#region scm, Rachel & Peng https://github.com/Microsoft/vscode/issues/53892
 
+	export interface ReadonlySourceControl {
```

Comment:
How about:

- `SourceControlReference`
- `ActiveSourceControl`
- `VisibleSourceControl`
- ... hmmm

@jrieken For more ideas.

Reply:
Why not just hand out references to `SourceControl` straight away? Are we scared that another extension does bag things with it?

Reply:
Yeah, those instances are the gate way to create groups, change group resources, etc... and there would be no way of preventing extensions from doing that.

In order to get that kind of functionality, I'd prefer opening up an official Git extension API, which I already started doing.